wagtail/contrib/search_promotions/models.py

```diff
@@ -66,6 +101,16 @@ def get_most_popular(cls, date_since=None):
 
 
 class QueryDailyHits(models.Model):
+    """
+    Model to query search promotion hits for a date.
+
+    Attributes
+
+       - query (models.ForeignKey): Matching query object.
+       - date (models.DateField): Matching date.
+       - hits = (models.IntegerField): Number of matches.
+    """
+
```

Comment:
```suggestion
```

This code is not changed at all by this PR, no need to change any docstrings here.